modules/oadp-configuring-aws-bsl.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-bsl.adoc_{context}"]
+= Configuring the backup storage location using AWS
+
+You can configure the {aws-short} backup storage location (BSL) as shown in the following example.
```

Comment:
Nit - not sure this is a big deal but maybe saying example procedure would be clearer:
```suggestion
You can configure the {aws-short} backup storage location (BSL) as shown in the following example procedure.
```

Up to you though